R/Task.R

```diff
@@ -70,10 +50,27 @@
 #'   Should sanity of data be checked initially at task creation?
 #'   You should have good reasons to turn this off (one might be speed).
 #'   Default is \code{TRUE}.
+#' @param censoring [\code{character(1)}]\cr
```

Comment:
why not doc these params on the page of the task that they refer to? so here: SurvTask